docs/_static/indsearch.js

```diff
@@ -55,7 +58,10 @@ function makeKeywordLabel(ind) {
 function makeVariableList(ind) {
     /* Print list of variables and include mouse-hover tooltip with variable description. */
     return Object.entries(ind.vars).map((kv) => {
-        const tooltip = `<button class="indVarname" title="${kv[1]}" alt="${kv[1]}">${kv[0]}</button>`;
+        /* kv[0] is the variable name, kv[1] is the variable description. */
+        /* Convert kv[1] to a string literal */
+        const text = escapeHTML(kv[1]);
+        const tooltip = `<button class="indVarname" title="${text}" alt="${text}">${kv[0]}</button>`;
```

Comment:
Looks good, we should probably use the function elsewhere too, just as a safety check. 

For example, `j <i --> test [rest of doc...]` renders as "j _test [rest of doc...]_"